src/win/mman.h

```diff
@@ -0,0 +1,76 @@
+/* This code was adapted from GitHub: https://github.com/alitrack/mman-win32 */
+
```

Comment:
While MIT license may be OK, simply copying this is a violation of the license terms.

Reply:
I added the MIT licence to the header and credited the original author, and the owner of the GitHub mirror/fork of the original work.